src/networks.js

```diff
@@ -0,0 +1,55 @@
+import env from './environment'
+
+const DEFAULT_IPFS_GATEWAY = 'https://ipfs.eth.aragon.network/ipfs'
+const DEFAULT_ORG_LOCATION = 'agreement3.aragonid.eth'
+
+const NETWORK_ENVIRONMENTS = new Map([
+  [
+    'ethereum',
+    {
+      chainId: 1,
+      legacyNetworkType: 'main',
+      ensAddress: '',
+      nodes: {
+        defaultEth: 'https://mainnet.eth.aragon.network/',
```

Comment:
I saw your comment in https://github.com/aragon/aragon/pull/1485 as to naming here, but I wonder if we should name this as "ethereum" if we are passing it down into the Connect configuration?

Reply:
endpoints.ethereum? yes I totally prefer it, was trying to stick to our current standards and forgot about your PR 🙈